doc/languages-frameworks/javascript.section.md

```diff
@@ -209,6 +209,7 @@ In the default `installPhase` set by `buildNpmPackage`, it uses `npm pack --json
 * `npmPackFlags`: Flags to pass to `npm pack`.
 * `npmPruneFlags`: Flags to pass to `npm prune`. Defaults to the value of `npmInstallFlags`.
 * `makeWrapperArgs`: Flags to pass to `makeWrapper`, added to executable calling the generated `.js` with `node` as an interpreter. These scripts are defined in `package.json`.
+* `nodejs`: The nodejs package to build the npm package. This in turns uses the corresponding npm package shipped with the nodejs chosen.
```

Comment:
```suggestion
* `nodejs`: The `nodejs` package to build against, using the corresponding `npm` shipped with that version of `node`. Defaults to `pkgs.nodejs`.
```

Maybe that might be slightly clearer?

Idk I'm open to alternative wordings too, it's just using "npm package" in two separate ways in this phrasing in confusing and it doesn't clearly state that it has a default value

Thanks again!

Reply:
happy to go with that, I'll commit this then will just squash the commits into one upon merging if that works for you.